### ISSUE TITLE
Multi-Modal Loss Function and Training Function

### ISSUE THREAD

**Issue by gvigkannan** [jeremykawahara/derm7pt#5]
Hello!
I hope this message finds you well.
Your paper on Multi-Modal Learning for Skin Lesion Detection and Classification is quite impressive and would love to know more about your implementation of the loss function mentioned in the paper. Could you please provide the code for the same?
 Thank you for your time!
